_posts/2021-04-14-accessing-the-relay-store.markdown

```diff
@@ -0,0 +1,260 @@
+---
+layout: epic
+title: "A Cool Way to Access the Relay Store"
+date: 2021-04-14
+categories: [relay, graphql, react, redis, tooling]
+author: anna
+---
+
+Need to access the Relay store without using a mutation? Relay's `commitLocalUpdate` allows you to modify the relay
+store from inside any relay component. The solution is particularly well suited to situations that just require
+client-side updates and do not involved changes to server-side data.
+
+<!-- more -->
+
+```js
+commitLocalUpdate(relay.environment, (store) => {
```

Comment:
As a sort of nice-to-have for the reader we should add a line just above this that includes the import:

```tsx
import { commitLocalUpdate} from 'relay-runtime' 
```